<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEditTtRssAccount</class>
 <widget class="QDialog" name="FormEditTtRssAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="7" column="0">
      <widget class="QPushButton" name="m_btnTestSetup">
       <property name="text">
        <string>&amp;Test setup</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QGroupBox" name="m_gbAuthentication">
       <property name="toolTip">
        <string>Some feeds require authentication, including GMail feeds. BASIC, NTLM-2 and DIGEST-MD5 authentication schemes are supported.</string>
       </property>
       <property name="title">
        <string>Authentication</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Username</string>
          </property>
          <property name="buddy">
           <cstring>m_txtUsername</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="buddy">
           <cstring>m_txtPassword</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="LineEditWithStatus" name="m_txtPassword" native="true"/>
        </item>
        <item row="0" column="1">
         <widget class="LineEditWithStatus" name="m_txtUsername" native="true"/>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="m_checkShowPassword">
          <property name="text">
           <string>Show password</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QGroupBox" name="m_gbHttpAuthentication">
       <property name="toolTip">
        <string>Some feeds require authentication, including GMail feeds. BASIC, NTLM-2 and DIGEST-MD5 authentication schemes are supported.</string>
       </property>
       <property name="title">
        <string>Requires HTTP authentication</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Username</string>
          </property>
          <property name="buddy">
           <cstring>m_txtHttpUsername</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="LineEditWithStatus" name="m_txtHttpUsername" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="buddy">
           <cstring>m_txtHttpPassword</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="LineEditWithStatus" name="m_txtHttpPassword" native="true"/>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="m_checkShowHttpPassword">
          <property name="text">
           <string>Show password</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="m_lblTitle">
         <property name="text">
          <string>URL</string>
         </property>
         <property name="buddy">
          <cstring>m_txtUrl</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LineEditWithStatus" name="m_txtUrl" native="true"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="m_lblDescription">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="m_lblServerSideUpdateInformation">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="m_checkServerSideUpdate">
       <property name="text">
        <string>Force execution of server-side update when updating feeds from RSS Guard</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="m_checkDownloadOnlyUnreadMessages">
       <property name="text">
        <string>Download only unread messages</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="LabelWithStatus" name="m_lblTestResult" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>68</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelWithStatus</class>
   <extends>QWidget</extends>
   <header>labelwithstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_checkDownloadOnlyUnreadMessages</tabstop>
  <tabstop>m_checkServerSideUpdate</tabstop>
  <tabstop>m_checkShowPassword</tabstop>
  <tabstop>m_gbHttpAuthentication</tabstop>
  <tabstop>m_checkShowHttpPassword</tabstop>
  <tabstop>m_btnTestSetup</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
